distributed/dashboard/templates/worker-table.html

```diff
@@ -27,6 +28,7 @@
         <td> </td>
```

Comment:
My guess is that we would want to conditionally add a `style=` keyword to the `tr` block just above this.

I'm also not an expert in Jinja, but a bit of stack overflow searching lead to https://stackoverflow.com/a/47610447 which might be enough?